aur.py

```diff
@@ -61,7 +73,7 @@ def main():
     module = AnsibleModule(
         argument_spec={
             'name': {
-                'required': True,
+                'required': False,
```

Comment:
This needs checking in `install_packages`, then.

Reply:
thanks for the feedback, changes have been made